DocNet.Console/Program.cs

```diff
@@ -1,14 +1,146 @@
 ﻿using log4net;
+using CommandLine;
+using System;
+using System.IO;
 
 namespace DocNet.Console
 {
-    public class Program
+    //using System;
+
+    public static class Program
     {
         private static readonly ILog Log = LogManager.GetLogger(typeof(Program));
 
+        //Command Line Options
+        public class Options
+        {
+            [Option('i', "input", Required = true, HelpText = "File/Folder to convert to documentation.")]
+            public string InputFile { get; set; }
+            [Option('o', "output", Required = true, HelpText = "Location to output documentation.")]
+            public string OutputFile { get; set; }
+            [Option('r', "recurse", HelpText = "Recursively search through input folder", DefaultValue = false)]
+            public bool RecurseOption { get; set; }
+            [Option('h', "help", HelpText = "Display command line help information", Required = false, DefaultValue = false)]
+            public bool Help { get; set; }
+        }
+
+        //Checks that command line input is correct and outputs a custom message depending on whats missing
+        static private bool CmdCheck(string[] args)
+        {
+            //Determine which required commands are missing
+
+            if (args.Length == 0)
```

Comment:
Might want to double-check the logic here. I think args[] might contain the entire command as the first argument (including the name of the program).